### PR TITLE
Fix Landscape mode, when menu pushed in Create mode

### PR DESCRIPTION
**Pull request by vladest**
Test plan:
- in Tablet in desktop mode click on Create button
- check Tablet switched to Landscape
- on Desktop menu select Settings->General (Audio etc)
- check if Tablet switched back to portrait mode

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/2083/HighFidelity-Beta-PR11071-45934cb53e2b1622ef3a0366f6703b68f563ca33.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2083/HighFidelity-Beta-PR11071-45934cb53e2b1622ef3a0366f6703b68f563ca33.exe

**Comment by YuppyPlays** [comment]
Testing.

**Comment by YuppyPlays** [comment]
Testing complete:

in Tablet mode click on Create button **Check**
check Tablet switched to Landscape **Check**
click on List, Properties tabs **Check**
check if content is not white **Check**
on Desktop menu select Settings->General (Audio etc)  **Check**
check if Tablet switched back to portrait mode **Check**

**Pass**

File: interface/src/ui/overlays/Web3DOverlay.cpp [hunk 1]
```diff
@@ -205,6 +205,8 @@ void Web3DOverlay::loadSourceURL() {
             _webSurface->getSurfaceContext()->setContextProperty("SoundCache", DependencyManager::get<SoundCache>().data());
 
             _webSurface->getSurfaceContext()->setContextProperty("pathToFonts", "../../");
+            _webSurface->getSurfaceContext()->setContextProperty("Paths", DependencyManager::get<PathUtils>().data());
```
Comment (ZappoMan) [inline interface/src/ui/overlays/Web3DOverlay.cpp]: @jherico - thoughts on this change?

Comment (vladest) [inline interface/src/ui/overlays/Web3DOverlay.cpp]: its already implemented in another PR, so I'll remove this

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/2105/HighFidelity-Beta-PR11071-3c9ab36416a187251a22947358a522dfdc4e6ace.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2105/HighFidelity-Beta-PR11071-3c9ab36416a187251a22947358a522dfdc4e6ace.exe

**Comment by anshuman64** [comment]
QA'ed again after some code changes. Still QA Pass. Just needs code review

**Pull request merged**